inc/template-functions.php

```diff
@@ -133,7 +81,9 @@ function nightingale_add_class_to_excerpt( $excerpt ) {
  * @param int $length length to shorten content to.
  */
 function nightingale_shorten_excerpt( $length ) {
-	if ( is_admin() ) return $length;
+	if ( is_admin() ) {
+		return $length;
+	}
 	return 20;
```

Comment:
That's a bit odd to hardcode in value 20 in the function? Suggested to be a default_length parameter? or const?
```nightingale_shorten_excerpt( $length, $default_length=20 ) {```

Reply:
I agree. This snippet was actually requested by the review team to be added verbatim. Seems very strange to me, but gives the desired output so not arguing...